RELEASE.md

```diff
@@ -1,27 +1,33 @@
-= Releasing the Jaeger Operator for Kubernetes
+# Releasing the Jaeger Operator for Kubernetes
 
 1. Make sure the `jaeger.version` file is up to date
 
-1. Prepare a changelog and get it merged. A list of commits since the last release (`v1.8.0` in the following example) can be obtained via:
+2. Prepare a changelog and get it merged. A list of commits since the last release (`v1.8.0` in the following example) can be obtained via:
```

Comment:
Markdown can't keep up with the counter?

Reply:
No - the blocks caused it problems, so it just reset to 1. I tried indenting the blocks to see if it would consider them part of the list item, but didn't work either.